package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "userland-migrations",
+  "version": "1.0.0",
+  "description": "A collection of migration recipes for userland code.",
+  "scripts": {
+    "test:e2e": "node --no-warnings --experimental-test-coverage --test-reporter=lcov --test-reporter-destination=./coverage.lcov --test-reporter=spec --test-reporter-destination=stdout --import './build/snapshots.ts' --test --test-coverage-include='recipes/**/*' --test-coverage-exclude='**/*.e2e.m(j|t)s' './packages/*/*.e2e.m(j|t)s'",
+    "lint": "biome lint ./",
+    "lint:fix": "biome lint --fix ./",
+    "format": "biome format ./",
```

Comment:
What's the difference between lint and format?

Reply:
if you take look to biome rules set there are two part linter and formater
- linter is a code analyser 
- formater is a code formater 

IDK if you get the idea put it's two separate things